00_hello_world/main.go

```diff
@@ -9,5 +9,20 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var firstNumber = 1
+	var secondNumber = 1
```

Comment:
You can fold this into one line: `firstNumber, secondNumber := 1, 1`. I would also suggest shorter variable names. `a` and `b` would be just fine in this context.

Reply:
Done.